ironfish/src/wallet/walletdb/walletdb.ts

```diff
@@ -761,6 +761,39 @@ export class WalletDB {
     await this.pendingTransactionHashes.clear(tx, account.prefixRange)
   }
 
+  async cleanupDeletedAccount(accountId: string, signal?: AbortSignal): Promise<void> {
```

Comment:
This duplicates most of the code in `cleanupDeletedAccounts`. Could we update one to call the other and avoid the duplication?